types/GasPriceOracle.md

```diff
@@ -0,0 +1,128 @@
+This is an example template for a new Query type. Every `Query` specification must include the following:
+
+
+## Type Name
+
+`Gas Price Oracle`
+
+
+## Description
+
+This is a proposal for a new query type for reporting a gas price to the Tellor network. The query type will be called `GasPriceOracle`. Its main use case is for gas refunds: protocols will need an accurate estimate for the cost of historical user transactions in gas. Given its approach, this `GasPriceOracle` is best suited for refunding users an approximation of their gas spent according to an average of gas prices from that time period.
+
+
+## Query Parameters
+
+The parameters of the `GasPriceOracle` query type will be the `chaindId` and `timestamp`
+
+```
+1. chainId
+    - description: unique id of the EVM chain (ex: rinkeby is 4)
+    - value type: `uint256`
+2. timestamp
+    - description: the unix timestamp to calculate on-chain gas price at
+    - value type: `uint256`
+```
+
+
+## Response Type
+
+`GasPriceOracle`'s response type is an unpacked 256 bit value with 18 decimals of precision:
+```
+- abi_type: ufixed256x18 (18 decimals of precision)
+- packed: false
+```
+
+
+## Query Data
+
+Query data is used to form your new Query's unique identifier, or query ID, and it's also included in emitted contract events so Tellor users and reporters can programmatically construct query objects.
+
+To generate the query data for an instance of your new Query type, first UTF-8 encode the parameter values in the order specified above. Then encode those `bytes` with the Query's type string.
+
+For example, to get the query data of an example instance of a `GasPriceOracle` query using Solidity:
+```s
+queryData = abi.encode("GasPriceOracle", abi.encode("1", "1650465649"))
```

Comment:
Looks like you're encoding the params as type `string`, when they're `uint256`. I think something like:
```s
queryData = abi.encode("GasPriceOracle", abi.encode(1, 1650465649)
```

Reply:
good catch